menus/significant-other.cson

```diff
@@ -4,7 +4,8 @@
     'submenu': [
       'label': 'Significant Other'
       'submenu': [
-        { 'label': 'Toggle', 'command': 'significant-other:toggle' }
+        { 'label': 'Toggle', 'command': 'significant-other:toggle' },
+        { 'label': 'Clear Cache', 'command': 'significant-other:clear-cache' }
```

Comment:
If Matchmaker was wrong, and you cache a wrong file, you have no option to reset the cache. I think the user should be able to clear cache if we did the mistake and put a wrong file in a cache.